cryptol-remote-api/python/cryptol/synchronous.py

```diff
@@ -0,0 +1,176 @@
+"""A synchronous, typed interface for the Cryptol bindings"""
+
+from __future__ import annotations
+
+import sys
+from typing import cast, Any, Optional, Union, List, Dict, TextIO
+from typing_extensions import Literal
+
+from . import solver
+from . import connection
+from . import cryptoltypes
+from .commands import *
+from . import CryptolConnection, SmtQueryType
+
+
+def connect_sync(command : Optional[str]=None,
```

Comment:
Do these functions need the `_sync` prefix, given that they're in their own module? (i.e., if you wanted to import the old and new interface, couldn't you have a prefix on the module imports to tell them apart?)

It's not a big deal, but it seems like if we move towards having the synchronous interface be the default it might be funny to have `_sync` on the methods. If anything we'd want the default interface to be smooth/streamlined and the odd one to have warts (if it has to -- i.e., if import prefixes aren't sufficient).

(Maybe I'm not seeing a reason to have the suffixes though?)

Reply:
I refactored so now the tests use `cryptol.sync.connect(...)` instead of `cryptol.connect_sync(...)`. I don't know if the module trickery I did to get `sync` working as an abbreviation is very Pythonic though... see `__init__.py`